README.md

```diff
@@ -230,42 +241,83 @@ Example response body:
   "dateToHMRCSubsidyUsage": "2021-07-19"
 }
 ```
-## Monitoring
+### `GET             /retrieve-exchange-rate/:date`
+This endpoint facilitates the retrieval of exchange rates for a specific date. Specifically, it aims to obtain the exchange rate for the last day of the previous month corresponding to the provided date.
 
-The following grafana and kibana dashboards are availble for this service
-* [grafana](https://grafana.tools.production.tax.service.gov.uk/d/RwwxDLSnz/eu-subsidy-compliance-frontend)
-* [kibana](https://kibana.tools.production.tax.service.gov.uk/app/kibana#/dashboard/eu-subsidy-compliance-frontend)
+Usage Example:
 
-## Runbook
+If the input date is 15/08/2023, the endpoint will retrieve the exchange rate for 31/07/2023.
 
-The runbook for this service can be found
-[here](https://confluence.tools.tax.service.gov.uk/display/SC/Runbook+-+Subsidy+Compliance).
+Process Flow:
 
+The endpoint initiates a check in the MonthlyExchangeRate cache for the required data.
+If the data is found in the cache, it is retrieved directly.
+If the data is not found in the cache, the module fetches it from the europa endpoint.
+A 200 (OK) response is returned upon successful retrieval of the correct date's exchange rate.
 
-## Formatting
-This is governed by the **.scalafmt.conf** file.
+Example response body:
+```json
+{
+  "currencyIso": "GBP",
+  "refCurrencyIso": "EUR",
+  "amount": 0.864,
+  "dateStart": "2023-07-01T00:00:00.000+00:00",
+  "dateEnd": "2023-07-31T00:00:00.000+00:00",
+  "createDate": "2024-02-02T00:00:00.000+00:00"
```

Comment:
Correct me if I'm wrong but originally the date format was `yyyy-MM-dd`

Reply:
Have amended this :)